lib/dbservice_web/endpoint.ex

```diff
@@ -44,5 +44,12 @@ defmodule DbserviceWeb.Endpoint do
   plug Plug.MethodOverride
   plug Plug.Head
   plug Plug.Session, @session_options
+
+  plug APIacFilterIPWhitelist,
+    whitelist: [
+      "127.0.0.1/32",
+      "35.190.11.178/32"
```

Comment:
is it possible to whitelist domains? instead of IPs? I think it would be better if we can control the domain names. They would be easy to read and update if needed

Reply:
Whitelisting domain was a priority for me too but unfortunately after a lot of research, I was unable to find out any trace of domain whitelisting in Elixir...